src/storage/admin/AdminTask.h

```diff
@@ -102,20 +108,27 @@ class AdminTask {
 
   virtual void cancel() {
     FLOG_INFO("task(%d, %d) cancelled", ctx_.jobId_, ctx_.taskId_);
+    canceled_ = true;
     auto suc = nebula::cpp2::ErrorCode::SUCCEEDED;
     rc_.compare_exchange_strong(suc, nebula::cpp2::ErrorCode::E_USER_CANCEL);
   }
 
+  virtual bool isRunning() { return isRunning_; }
+
+  virtual bool isCanceled() { return canceled_; }
+
   meta::cpp2::AdminCmd cmdType() { return ctx_.cmd_; }
 
  public:
   std::atomic<size_t> unFinishedSubTask_;
   SubTaskQueue subtasks_;
+  std::atomic<bool> isRunning_{false};
```

Comment:
keep consistent naming with canceled? call running_ and canceled , or isRunning and isCanceled